slangpy/core/calldata.py

```diff
@@ -33,6 +33,11 @@
 
 _DUMP_GENERATED_SHADERS = False
 _DUMP_SLANG_INTERMEDIATES = False
+_PRINT_GENERATED_SHADERS = os.environ.get("SLANGPY_PRINT_SHADERS", "false").lower() in (
```

Comment:
should the env variable be named `SLANGPY_PRINT_GENERATED_SHADERS` to be more consistent with the `set_print_generated_shaders`?